components/Footer/Footer.tsx

```diff
@@ -1,14 +1,20 @@
 import Image from 'next/image';
 import styles from './Footer.module.scss';
+import type FooterProps from './types';
 
-export const Footer = () => (
+function getYear(date: string) {
+    const dateObject = new Date(date);
+    return dateObject.getFullYear();
+}
+
+export const Footer = ({ last_published_at }: FooterProps) => (
```

Comment:
Nitpick: I typically use camelCase for variables being passed into react components, e.g.
```suggestion
export const Footer = ({ lastPublishedAt }: FooterProps) => (
```